docs/navigation-advanced.md

```diff
@@ -0,0 +1,399 @@
+# Advanced App navigation
+This documentation describes the steps to modify the NavigationService to allow navigation in different frames and  navigation levels to support advanced navigation scenarios, like the following:
+
+**App launching**
+ - Show a Startup page on app launching and navigate to a navigation pane shell page from there. 
+ - Give the possibility to go back to the startup page or not.
+ 
+**NavigationPane Scenarios**
```

Comment:
Question: Does this doc cover all of these scenarios? Is it possible (easy) to implement whatever is needed if someone only wanted to do one/some of them? Or is the logic all tied together?